src/module/data/actor-character.mjs

```diff
@@ -16,12 +16,10 @@ export default class GrimwildCharacter extends GrimwildActorBase {
 			min: 0
 		});
 
-		schema.healing = new fields.SchemaField({
-			value: new fields.NumberField({
-				...requiredInteger,
-				initial: 10,
-				min: 0
-			})
+		schema.level = new fields.NumberField({
```

Comment:
I would make this one a derived attribute or getter since it's tied directly to XP.